### ISSUE TITLE
Add an option to only allow specific affiliates to be selected at checkout

### ISSUE THREAD

**Issue by michaelbeil** [AffiliateWP/affiliatewp-checkout-referrals#26]
We've received the following feature request:

>It would be a good condition if customer could select only specific affiliates from the checkout selection i.e. charities, and not see all registered affiliates.

>I couldn't find an option for specifying the affiliate items of the checkout selection!

Ticket: https://secure.helpscout.net/conversation/781105587/0/?folderId=205524.

**Comment by michaelbeil** [comment]
A [customer sent in a ticket](https://secure.helpscout.net/conversation/1036232655/140425/) with a summary of **Is there a way to show only select affiliates here?**:
>We would like to use this extension to only show a select set of affiliates in the dropdown, not all of them.

>Is it possible to show only some that we can select and not all affiliates in the dropdown during checkout? If not, what would it take to get this done?

I let them know that we may add this in the future, but it would require custom development now.

**Comment by DrewAPicture** [comment]
PR: #32 introduces a filter on the arguments used to retrieve affiliates.

**Comment by DrewAPicture** [comment]
Merged #32 to introduce the `affwp_checkout_referrals_get_affiliates_args` filter.

In < AffiliateWP 2.4.3, to exclude affiliates:
```php
<?php
add_filter( 'affwp_checkout_referrals_get_affiliates_args', function( $args, $context ) {
	$args['exclude'] = array( 1, 2, 3 );

	return $args;
}, 10, 2 );
```

In AffiliateWP 2.4.3+, support is being added for an `include` argument:
```php
<?php
add_filter( 'affwp_checkout_referrals_get_affiliates_args', function( $args, $context ) {
	$args['include'] = array( 1, 2, 3);

	return $args;
}, 10, 2 );
```
